lib/elixir/lib/code.ex

```diff
@@ -425,6 +501,12 @@ defmodule Code do
   In most cases, `ensure_loaded/1` is enough. `ensure_compiled/1`
   must be used in rare cases, usually involving macros that need to
   invoke a module for callback information.
+
+  ## Examples
+
+      iex> Code.ensure_loaded(Atom)
```

Comment:
Nice! Let's also add an example when a module that does not exist (it can be called DoesNotExist) is called too.